tests/phpunit/Mutator/ProfileListProvider.php

```diff
@@ -103,7 +103,7 @@ public static function implementedMutatorProvider(): Generator
                 continue;
             }
 
-            if (!self::extendsMutator($mutatorReflection)) {
+            if (!$mutatorReflection->implementsInterface(Mutator::class)) {
```

Comment:
although this does not change anything in practice for now, this is still more correct as if a new mutator does not extend the util one, it would not be collected